src/kactl/strings/AhoCorasick.h

```diff
@@ -1,4 +1,5 @@
 /**
+ * Name: Aho-Corasick algorithm
```

Comment:
`Aho--Corasick` :wink: ; also I would omit `algorithm` since we do it in Edmonds--Karp, for example